### PR TITLE
Add a GitHub Case Study Write-Up

### PR DESCRIPTION
**Pull request by Julian**
~~This is waiting for a last section to be plopped in (which I'm porting from the case study document)~~ EDIT: it's in, and then also needs a second look-over (and of course a review from others with any comments).

But putting it up now, will continue to tweak, but feel free to chime in with comments!

(The title is/was also a placeholder, not sold on it as-is)

### CODE REVIEW

File: pages/posts/github-case-study.md
Comment (Julian) [inline pages/posts/github-case-study.md:49]: The doc didn't attribute its contents to either of the team members at GitHub -- not sure if you know who the voice of the document is. For now I attributed the quote(s) to both of them, though I guess we can also possibly ping them to kindly review the post itself at some point if we want and if they care to.

File: pages/posts/github-case-study.md
Comment (Julian) [inline pages/posts/github-case-study.md:5]: This SVG needs shrinking probably, I guess to some extent it too was a placeholder.

File: pages/posts/github-case-study.md
Comment (Relequestual) [reply pages/posts/github-case-study.md:49]: Yes, we will do that for sure.

File: pages/posts/github-case-study.md
Comment (Relequestual) [reply pages/posts/github-case-study.md:5]: Prefer webp. You can download the cli converter from here: https://developers.google.com/speed/webp

Yeah, I think it is a placeholder. Will need something else.

**Review by Relequestual — COMMENTED** [review]
Looks like a great start! Thanks!

I feel that some of the content goes into too much depth, but I don't know if my target readership is right either.

My preference would be to convert the technical detail into business value, which speaks to both developers and non-developers.

Of course, shout when you've made revisions and added that last section as you mentioned.
Then I'll reach out to the people at GitHub for commnet and review.
🙌

File: pages/posts/github-case-study.md
Comment (Julian) [reply pages/posts/github-case-study.md:5]: Surely webp is preferred over png/jpg/etc. but not for svg? Though if/when we have an idea for a replacement probably it won't matter.

**Comment by Julian** [comment]
Cool! Thanks for the review -- I actually added that section (it was the solution section) so will take out of draft, but will take another look anyhow now that you left some comments to see if there's anything else in the document that can be added!

> I feel that some of the content goes into too much depth, but I don't know if my target readership is right either.

Yeah not sure either, certainly the opinion is welcome. I'm very much personally not a fan of overspeaking business value when it's not clear there's good evidence for it, and for me the bar is very high there to me, so simply stating facts is always my preference unless there's specific evidence to speak of (e.g. "I did X and it made my machines faster and so I saved $Y" which is generally incontrovertible). I know others have different opinions certainly, so I'm clearly happy if we decide to go for more there, though I'm not sure how much more was in the source doc, will have another look.

File: pages/posts/github-case-study.md
Comment (Relequestual) [reply pages/posts/github-case-study.md:5]: You know, I missed that it was svg.
But anwyay, I expect it will be replaced. I... didn't realise what it was when I made that comment anyway.

**Comment by Relequestual** [comment]
> ...for me the bar is very high there to me, so simply stating facts is always my preference unless there's specific evidence to speak of

Oh for sure. I only use what is provided. No making up or exaggerating things here.

File: pages/posts/github-case-study.md
Comment (rachmari) [inline pages/posts/github-case-study.md:25]: I think this might be a typo.

```suggestion
More specifically, some page content is assembled fully automatically from JSON data files and some JSON is manually authored by content writers.
```

**Review by rachmari — COMMENTED** [review]
This looks wonderful! Thank you so much for the write up. I'm not sure that it's valuable, but the code used to power docs.github.com is fully open-source at https://github.com/github/docs. If folks want to poke around source code, it's all there including the content.

I'm working on getting the images from our marketing department. I can link those up here when I receive them.

File: pages/posts/github-case-study.md
Comment (Julian) [reply pages/posts/github-case-study.md:25]: Indeed, thanks!!

**Comment by rachmari** [comment]
@Relequestual I uploaded 3 images that were approved for use in the blog post to the Google drive you set up: 
https://drive.google.com/drive/folders/1xvTlmne0KJSA31-nPcW84ab8bBsTJFPC?usp=drive_link

**Comment by Relequestual** [comment]
I've added the images.
However, I've noticed a bug where the images in the article don't display in the deploy preview... which is pretty annoying. I'm not sure how we fix that, and a new website is coming soon so I'm not sure it's worthwhile fixing.

So, if you want to preview with the images in place, I can make a screenshot, or you can clone it and run it locally.

@rachmari I think we are just waiting on your approval now. I don't know if we want to coordinate publication with anything on the GitHub side like social media postings. (cc @benjagm)

**Comment by rachmari** [comment]
@Relequestual 

> @rachmari I think we are just waiting on your approval now. I don't know if we want to coordinate publication with anything on the GitHub side like social media postings.

The last person that needs to approve this is @mollyryner. I'll defer to Molly on whether we should time this with any GitHub-related posts.

File: pages/posts/github-case-study.md
Comment (gregsdennis) [inline pages/posts/github-case-study.md:60]: Is it necessary to restate Rachael's qualifications (see line 54).  Typically I see secondary quotes attributed using just the last name.  (We need a professional editor 😄.)

File: pages/posts/github-case-study.md
Comment (gregsdennis) [inline pages/posts/github-case-study.md:54]: Does the `<p className="text-2xl my-10">` match the styling of block quotes in other posts?  It'd be good to maintain consistency.

File: pages/posts/github-case-study.md
Comment (mollyryner) [inline pages/posts/github-case-study.md:74]: Can we please remove this sentence?

**Comment by mollyryner** [comment]
Thanks, I reviewed and left one comment. No need to coordinate with anything on our end. Once it goes live, we'll share with our social team for amplification.

File: pages/posts/github-case-study.md
Comment (Julian) [reply pages/posts/github-case-study.md:54]: It seems like yes, see e.g. https://github.com/json-schema-org/blog/blob/ae73ee9f108e8cf3a468898c4cbaf31e120351d3/pages/posts/tyler-technologies-case-study.md?plain=1#L52

File: pages/posts/github-case-study.md
Comment (Julian) [reply pages/posts/github-case-study.md:74]: Done!

File: pages/posts/github-case-study.md
Comment (Julian) [reply pages/posts/github-case-study.md:60]: I am of course no professional editor but my suspicion is (even besides style being a matter mostly of readability) that it might also differ given the first quote was a blockquote (as opposed to being truly inline in the text) which are more like "highlights" that in theory someone may have only glanced over. But clearly no idea, I'd probably leave it though unless you really think it matters.

Comment (gregsdennis) [reply pages/posts/github-case-study.md:60]: It just feels redundant to me to have it in both places. Leave it if you think it's fine.

**Review by gregsdennis — APPROVED** [review]

File: pages/posts/github-case-study.md
Comment (Relequestual) [reply pages/posts/github-case-study.md:60]: I chocse this pattern because it makes the block quotes more copyable. I've seen this done many times in other press release style articles.

Block quotes may be read before the rest of the article as someone scans it to see if it's worthwhile. IMHO, it shouldn't be considered really part of the article in terms of having all the context of the prior content.

**Review by Relequestual — APPROVED** [review]

**Review by mollyryner — APPROVED** [review]

**Comment by Julian** [comment]
Hooray, thanks all!! Exciting stuff.

**Comment by Julian** [comment]
Live at https://json-schema.org/blog/posts/github-case-study!

**Pull request merged**
